packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -306,14 +342,21 @@ class Plugin {
   final String name;
   final String path;
 
+  /// The name of the interface package that this plugin implements.
+  final String implement;
```

Comment:
Consider a more descriptive name. "implementsPackage"? "packageImplements"? not quite sure.

If this doesn't implement a package, is it null?

Reply:
Done